### DISCUSSION TITLE
Mixing mesh elements

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#22122] [Q&A Meshing] [answered]
This may be a simple questions that is already answered somewhere. Is it possible to use a mesh with different elements (eg tets and hexes) but arranged so that each block only contains one kind of mesh elements, but different blocks can have different kinds of elements? I know that within a block, MOOSE seems not to like mixed elements.

**Comment by GiudGiud** [comment, marked as answer]
yes it s very much supported

just a detail but libmesh as well does not like mixed elements

**Comment by heinono1** [comment]
Thanks! I thought it might be, but I wanted to make sure.
